CHANGES.rst

```diff
@@ -1,39 +1,61 @@
 Changes
 =======
 
-Version 0.5.0 (2019-04-24)
+Version 0.6.0 (2019-12-27)
 --------------------------
 
 - Users:
-
-  - Allows to explore workflow results by running interactive Jupyter notebook
-    sessions on the workspace files.
-  - Allows to declare computing resources needed for workflow runs, such as
-    access to CVMFS repositories.
-  - Improves ``reana-client`` command-line client with new options to stop
-    workflows, diff workflows, move and remove files.
-  - Upgrades CWL engine to 1.0.20181118133959.
-  - See additional changes in `reana-client release notes <https://reana-client.readthedocs.io/en/latest/changes.html#version-0-5-0-2019-04-24>`_.
+    - Adds support for HTCondor compute backends for all workflow engines.
+    - Adds support for Slurm compute backends for all workflow engines.
+    - Allows to run hybrid analysis pipelines when various parts of the workflow run on various HTCondor, Kubernetes and Slurm compute backends.
```

Comment:
What do you think about this or similar? (just removing `various` dup, adding more punctuation and links):
"Allows running hybrid analysis pipelines, where different parts of the workflow can run on various compute backends: [HTCondor](https://research.cs.wisc.edu/htcondor/), [Kubernetes](https://kubernetes.io/) and [Slurm](https://slurm.schedmd.com/documentation.html)."

Reply:
Yes, it reads better, reformulated -- but kept two "different" occurrences for wanted stylistic effect